### PR TITLE
Update Bundler/Rubocop

### PR DESCRIPTION
**Pull request by tpendragon**

### CODE REVIEW

File: lib/cicognara/book_indexer.rb [hunk 1]
```diff
@@ -6,7 +6,7 @@ def initialize(books)
     end
 
     def to_solr
-      @to_solr ||= marc_solr.present? ? marc_solr : blank_records
+      @to_solr ||= marc_solr.presence || blank_records
```
Comment (hackmastera) [inline lib/cicognara/book_indexer.rb:9]: This is cool, didn't know about it

File: app/helpers/advanced_helper.rb [hunk 2]
```diff
@@ -26,18 +26,17 @@ def advanced_key_value
 
   # carries over original search field and original guided search fields if user switches to guided search from regular search
   def guided_field(field_num, default_val)
-    if field_num == :f1 && params[:f1].nil? && params[:f2].nil? && params[:f3].nil? && params[:search_field] && search_fields_for_advanced_search[params[:search_field]]
-      return search_fields_for_advanced_search[params[:search_field]].key || default_val
-    end
+    return search_fields_for_advanced_search[params[:search_field]].key || default_val if field_num == :f1 && params[:f1].nil? && params[:f2].nil? && params[:f3].nil? && params[:search_field] && search_fields_for_advanced_search[params[:search_field]]
```
Comment (hackmastera) [inline app/helpers/advanced_helper.rb:29]: Could we move the clause of this conditional to a method that's called what it's checking for?

File: .rubocop.yml [hunk 3]
```diff
@@ -93,3 +94,14 @@ Style/SafeNavigation:
 
 Style/RegexpLiteral:
   Enabled: false
+
+Security/Open:
+  Exclude:
+    - 'app/services/getty_parser.rb'
```
Comment (hackmastera) [inline .rubocop.yml:100]: Checks for `Kernel/open` but I don't see it, do you??

Comment (tpendragon) [reply .rubocop.yml:100]: Yeah. https://github.com/pulibrary/cicognara-rails/blob/f4ccc90e5359d3bf69f7dba46af119af4a0204bd/app/services/getty_parser.rb#L56

File: lib/cicognara/book_indexer.rb (hunk 1 above)
Comment (tpendragon) [reply lib/cicognara/book_indexer.rb:9]: I didn't either. Thanks Rubocop.

File: app/helpers/advanced_helper.rb (hunk 2 above)
Comment (tpendragon) [reply app/helpers/advanced_helper.rb]: Done.

**Comment by hackmastera** [comment]
hm, tests aren't happy with that refactor though

**Pull request merged**